public/js/src/module/application-cache.js

```diff
@@ -14,7 +14,7 @@ function init( survey ) {
                     // Registration was successful
                     console.log( 'Offline application service worker registration successful with scope: ', registration.scope );
                     setInterval( () => {
-                        console.log( 'Checking for offlince application cache service worker update' );
+                        console.log( 'Checking for offline application cache service worker update' );
                         registration.update();
```

Comment:
Am I right that these periodic checks still benefit from browser/HTTP caching, so long as the gap between `update()` calls does not exceed 24 hours?

> The `update()` method of the `ServiceWorkerRegistration` interface attempts to update the service worker. It fetches the worker's script URL, and if the new worker is not byte-by-byte identical to the current worker, it installs the new worker. The fetch of the worker bypasses any browser caches if the previous fetch occurred over 24 hours ago. (https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration/update)

Reply:
Good point!

If the service worker (offline-app-worker.js) itself is cached by the browser, I think you're right that the HTTP cache would be used. Probably no real benefit because this happens asynchronously anyway, I believe.

It would probably be best not to include that offline-app-worker.js in any HTTP cache but thankfully as you discovered, the clever spec folks thought about that, so the worst thing that could happen would be a 24 hr delay in updating the app (cache).